features/FEATURE_LWIP/TESTS/mbedmicro-net/tcp_echo_parallel/main.cpp

```diff
@@ -20,6 +20,8 @@
 #define MBED_CFG_TCP_CLIENT_ECHO_THREADS 3
 #endif
 
+#define PARALLEL_STACK_SIZE 1024
```

Comment:
Nit: This should have an ifdef guard to allow users to override the setting from compilation.